GETTING_STARTED_ADD_PAT_OWNER.md

```diff
@@ -0,0 +1,184 @@
+        * Select Create
+
+    * Create new repository by fetching source code from Cobalt's master project
+        * Select Repos tab within side-navigation menu
+        * Select 'Import a repository' from the Repos tab sub-menu
+        * Enter the Cobalt Clone URL (https://github.com/microsoft/cobalt.git) and select Import
+
+            ![image](https://user-images.githubusercontent.com/10041279/63459072-8ec4cf00-c419-11e9-8ef4-ee7db827e49c.png)
+
+    * Initialize new Azure Devops pipeline
+        * Select Pipelines tab from within side-navigation menu
+        * Select Create Pipeline and then choose 'Azure Repos Git [YAML]'
+            ![image](https://user-images.githubusercontent.com/10041279/63459652-89b44f80-c41a-11e9-829a-05a6888b7673.png)
+        * Find and select the newly created repository from dropdown menu
+        * Import YAML by selecting 'Existing Azure Pipelines YAML file'
+            * Enter the path to the devops yaml file that lives within your newly created repo. (i.e. devops/providers/azure-devops/templates/azure-pipelines.yml)
+            ![image](https://user-images.githubusercontent.com/10041279/63459938-21b23900-c41b-11e9-9b9c-2dfa72e51350.png)
+            > NOTE: Automatic drop-down does not always populate with yaml file options. It may be necessary to simply copy and paste the above path.
+        * Review devops pipeline YAML and only keep templates relevant to your enterprise patterns.
+            * Remove jobName configurations not relevant to your enterprise patterns.
+                ```yaml
+                configurationMatrix:
+                - jobName: az_service_single_region
+                terraformTemplatePath: 'infra/templates/az-service-single-region'
+                terraformWorkspacePrefix: 'sr'
+                environmentsToTeardownAfterRelease:
+                - 'devint'
+                ```
+            > NOTE: If new to Cobalt, we recommend keeping the az_hello_world template as a starter template. This step can also be completed later as a code commit to your repo.
+        * Save and run
+            > NOTE: Azure Devops forces a run so expect this to fail. Future steps will solve this problem.
+
+2. Provision Azure resources needed for Azure Devops pipeline
+
+    * Create a registered Azure AD (AAD) app for Cobalt deployments
+        * Sign-in to your organization's Azure account. (https://portal.azure.com)
+        * Filter for Azure Active Directory and navigate to it's menu
+        * Select App registrations from the menu blade
+        * Click [Add/+] New registration then enter a name for the application (ex. cobalt-hw-admin-sp)
+        * Choose single tenant as a supported account type to keep things simple.
+        * Click Register
+
+    * Configure the new AAD app as a Cobalt admin service-principal/service-endpoint
```

Comment:
Need to describe addition of required permission:
* Azure Active Directory Graph
  * Application.ReadWrite.OwnedBy

Reply:
Also... must click [ Grant admin consent for CSEWhite ] to grant the permission to the SP